src/Common.cpp

```diff
@@ -214,6 +214,18 @@ std::string NAS2D::trimWhitespace(std::string string)
 	return string.substr(first_non_space, last_non_space - first_non_space + 1);
 }
 
+bool NAS2D::startsWith(const std::string& string, const std::string& start)
+{
+	auto found_loc = string.find(start);
+	return found_loc != std::string::npos && found_loc == 0;
+}
+
+bool NAS2D::endsWith(const std::string& string, const std::string& end)
+{
+	auto found_loc = string.rfind(end);
+	return found_loc != std::string::npos && found_loc == string.size() - end.size();
```

Comment:
Same as the above.

Reply:
This could use a bit of polish. Specifically only check the last N characters of the input string via `std::string::substr`.